Tests/fixtures/smartphone-19.yml

```diff
@@ -557,3 +557,399 @@
+    engine_version: ""
+  device:
+    type: tablet
+    brand: Lenovo
+    model: TAB4 8 Plus
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.0; Lenovo TB-7304X) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36
+  os:
+    name: Android
+    version: "7.0"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 86.0.4240.198
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tablet
+    brand: Lenovo
+    model: Tab 7 Essential
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.0; Lenovo TB-7504X) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.92 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "7.0"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 77.0.3865.92
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tablet
+    brand: Lenovo
+    model: Tab 7
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 4.4.2; Lenovo TAB 2 A7-30GC) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
+  os:
+    name: Android
+    version: 4.4.2
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 81.0.4044.138
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tablet
+    brand: Lenovo
+    model: TAB 2 A7
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 7.1.2; 15 Lite) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 YaApp_Android/20.110.0 YaSearchBrowser/20.110.0 BroPP/1.0 SA/3 Mobile Safari/537.36
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ARM
+  client:
+    type: browser
+    name: Yandex Browser
+    version: 20.110.0
+    engine: Blink
+    engine_version: ""
+  device:
+    type: smartphone
+    brand: Meizu
+    model: 15 Lite
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.2; Smart Box v009 Build/NHG47L; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/52.0.2743.100 Safari/537.36
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Webview
+    version: 52.0.2743.100
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Rombica
+    model: Smart Box v009
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 4.4.4; Leff Shell HV320WHB-N55) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 YaBrowser/18.11.1.1011.01 Safari/537.36
+  os:
+    name: Android
+    version: 4.4.4
+    platform: ""
+  client:
+    type: browser
+    name: Yandex Browser
+    version: 18.11.1.1011.01
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Leff
+    model: HV320WHB-N55
+  os_family: Android
+  browser_family: Unknown
```

Comment:
@sgiehl 

This definition bothers me.

Leff Shell HV320WHB-N55
Shell Prefix-means that the mainboard or panel number goes next
HV320WHB-N55  - Mainboard ID from the manufacturer BOE
![image](https://user-images.githubusercontent.com/1337066/101754134-74b67080-3ae4-11eb-86b8-4ee4c7654adb.png)

a similar mainboard is used in the brand Manta model 32lha19s, Digma DM-LED32R301BT2S etc

Leave the model empty?